quinn-proto/src/endpoint.rs

```diff
@@ -122,11 +122,13 @@ where
                     warn!("duplicate reset token");
                 }
             }
-            RetireConnectionId(seq) => {
+            RetireConnectionId(seq, allow_more_cid) => {
```

Comment:
Maybe call this `allow_more_cids`? Reads more natural to me.

Reply:
Or alternatively, `issue_new_cid`?